packages/cli/src/lib/cmds/build.ts

```diff
@@ -142,6 +142,25 @@ class Build {
         APP_BUNDLE_BUILD_OUTPUT_FILE_NAME, APP_BUNDLE_SIGNED_FILE_NAME);
   }
 
+  /**
+   * Based on the promptResponse to update the project or not, run an update or print the relevant warning message.
```

Comment:
Could you explain what the return value signifies?

Reply:
Added a @return to the method comment